### PR TITLE
Update favicon

### PR DESCRIPTION
**Pull request by djnunez-aot**
*Issue #355 :*
https://github.com/bcgov/met-public/issues/355

-update favicon with the bc gov logo(no background)

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the sbc-auth license (Apache 2.0).

### CODE REVIEW

File: met-web/public/index.html [hunk 1]
```diff
@@ -1,26 +1,20 @@
...
-  </body>
+    <head>
+        <meta charset="utf-8" />
+        <title>Modern Engagement Tool</title>
+        <meta name="viewport" content="width=device-width, initial-scale=1" />
+        <meta name="theme-color" content="#000000" />
+        <meta name="description" content="Web site of MET" />
+        <script src="%PUBLIC_URL%/config/config.js"></script>
+        <script src="/snowplow.js" charset="UTF-8"></script>
+        <link rel="icon" href="%PUBLIC_URL%/BCFavIcon.png" />
```
Comment (jadmsaadaot) [inline met-web/public/index.html:11]: I think it's better to have a favicon.ico file if it's easy. All modern browsers support .png, but maybe some public users are operating on weird browsers. And the name should be "favicon"

**Review by jadmsaadaot — CHANGES_REQUESTED** [review]
Just a consideration

**Review by jadmsaadaot — APPROVED** [review]

**Review by saravanpa-aot — COMMENTED** [review]
Did the order of links change intentionally ?
hard to decipher whats the actual change

**Comment by jadmsaadaot** [comment]
> Did the order of links change intentionally ? hard to decipher whats the actual change

**Comment by jadmsaadaot** [comment]
> Did the order of links change intentionally ? hard to decipher whats the actual change

I think <link rel="icon" href="%PUBLIC_URL%/BCFavIcon.png" /> was just added, prettier change spacing so that's why everything changed. But order is the same

**Review by tiagograf-AOT — APPROVED** [review]

**Pull request merged**